README.md

```diff
@@ -227,13 +227,15 @@ $ mkdir build && cd build
 $ cmake -G Ninja .. \
     -DCMAKE_BUILD_TYPE=RELEASE \
     -DOP_OPTIMIZATION_BENCHMARKS=ON \
+    -DCMAKE_CXX_COMPILER=clang++ \
```

Comment:
I use version 14.0.0-1ubuntu1.1 and I have encountered this problem: ![image](https://github.com/buddy-compiler/buddy-benchmark/assets/56058848/8db5eaa3-6616-4e21-a5b2-973e3b013d06). Is it related to clang version?

What's more, I don't think we I don't think users will pre-install clang in their machine(someone like me will directly use llvm/bin/clang++), so may be using `\path\to\clang++` is more appropriate.

Reply:
The clang version should match with mlir-translate, I'm using rolling release distribution so the version of /usr/bin/clang++ is 16. I will amend the markdown.

Reply:
Sorry for typo I mean llvm/build/bin/clang++ instead of llvm/bin/clang++. Actually I use `buddy-mlir/llvm/build/bin/clang++` and it is generated together with mlir-translate, so their versions can be matched. I will try the compilation process later.